frontend/src/metabase/visualizations/components/settings/ChartSettingFieldPicker.jsx

```diff
@@ -28,6 +34,17 @@ const ChartSettingFieldPicker = ({
       columnKey = getColumnKey(column);
     }
   }
+
+  let seriesKey;
+  if (series && columnKey && showColorPicker) {
+    const seriesForColumn = series.find(single => {
+      const tempColumn = single.data.cols[1];
```

Comment:
what's a `tempColumn`? Could the name be more descriptive?

Reply:
I'll rename this to metricColumn, since this column represents the metric in this single series